custom/icds_reports/templates/icds_reports/icds_app/kpi.directive.html

```diff
@@ -1,49 +1,51 @@
-                    <div class="kpi-justify-center">
-                        <span class="number white" ng-if="cell.format == 'div'"><span class="white" ng-bind="cell.value | indiaNumbers"></span>/<span class="white" ng-bind="cell.all | indiaNumbers"></span></span>
-                        <span class="number white" ng-if="cell.format == 'number'" ng-bind="cell.value | indiaNumbers"></span>
-                        <span class="string white" ng-if="cell.format == 'string'" ng-bind="cell.value"></span>
-                        <div class="kpi-numbers-format" ng-if="cell.format == 'percent_and_div'">
-                            <div class="number white"><span class="white" ng-bind="(cell.value / (cell.all || 1) * 100) | number:2"></span>%</div>
-                            <div class="second_number white">(<span class="white" ng-bind="cell.value | indiaNumbers"></span> / <span  class="white" ng-bind="cell.all | indiaNumbers"></span>)</div>
-                        </div>
-                        <div class="kpi-numbers-format" ng-if="cell.format == 'number_and_percent'">
-                            <div class="number white"><span class="white" ng-bind="cell.value | indiaNumbers"></span></div>
-                            <div class="second_number white"><span class="white" ng-bind="(cell.value / (cell.all || 1) * 100) | number:2"></span>%</div>
-                        </div>
-                    </div>
-                    <div class="kpi-justify-center" ng-if="cell.show_percent === void(0)"
-                         ng-class="{'kpi-percent': cell.redirect}">
-                        <span class="white" ng-class="{'kpi-percents-text': cell.redirect}"
-                              ng-show="cell.percent !== null && $ctrl.isNumber(cell.percent) && $ctrl.showPercentInfo()">
-                            <i class="fa" style="font-size: 1.5em" ng-show="cell.percent !== 0"
-                               ng-class="{'fa-arrow-down': cell.percent < 0, 'fa-arrow-up': cell.percent > 0, 'green': cell.color === 'green', 'red': cell.color === 'red'}" ></i>&nbsp;
-                            <span class="white" ng-bind="cell.percent | number:2"></span>% from previous {$ cell.frequency $}
-                        </span>
-                        <span class="white" ng-class="{'kpi-percents-text': cell.redirect}"
-                              ng-show="cell.percent !== null && !$ctrl.isNumber(cell.percent) && $ctrl.showPercentInfo()">
-                            <span class="white" ng-bind="cell.percent"></span>
-                        </span>
-                    </div>
-                    <div class="row see-more-content">
-                        <div class="col-md-offset-10 col-md-2"></div>
-                        <span ng-show="cell.redirect" class="see-more" aria-hidden="true"><a href="{$ $ctrl.goToStep(cell.redirect) $}" class="fa">See more</a></span>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </div>
```

Comment:
what changed here? the whitespace diff makes it a bit tricky to tell

Reply:
According to spec, `LSs launched` card should be a third indicator in program summary page. Usually, these indicators are sent as arrays of 2. If the new indicator (array with one element) is inserted, it looks like:
![Screenshot from 2020-05-21 18-03-38](https://user-images.githubusercontent.com/56248457/82559623-96bec480-9b8d-11ea-957b-bff114d30dcc.png)

To avoid this, I have moved the class `"row"` to the outer div

Reply:
hey, this is happening, I think because the way you inserted it into the response result list above. if you will add it properly from the backend. you probably wont need to update the code here